cluster-autoscaler/provisioningrequest/orchestrator/orchestrator_test.go

```diff
@@ -76,17 +87,50 @@ func TestScaleUp(t *testing.T) {
 			Class:    v1beta1.ProvisioningClassCheckCapacity,
 		})
 
-	// Active atomic scale up request.
+	// Active atomic scale up requests.
 	atomicScaleUpProvReq := provreqwrapper.BuildValidTestProvisioningRequestFromOptions(
 		provreqwrapper.TestProvReqOptions{
 			Name:     "atomicScaleUpProvReq",
-			CPU:      "1",
+			CPU:      "5m",
+			Memory:   "5",
+			PodCount: int32(5),
+			Class:    v1beta1.ProvisioningClassAtomicScaleUp,
+		})
+	largeAtomicScaleUpProvReq := provreqwrapper.BuildValidTestProvisioningRequestFromOptions(
+		provreqwrapper.TestProvReqOptions{
+			Name:     "largeAtomicScaleUpProvReq",
+			CPU:      "1m",
+			Memory:   "100",
+			PodCount: int32(100),
+			Class:    v1beta1.ProvisioningClassAtomicScaleUp,
+		})
+	impossibleAtomicScaleUpReq := provreqwrapper.BuildValidTestProvisioningRequestFromOptions(
+		provreqwrapper.TestProvReqOptions{
+			Name:     "impossibleAtomicScaleUpRequest",
+			CPU:      "1m",
 			Memory:   "1",
+			PodCount: int32(5001),
+			Class:    v1beta1.ProvisioningClassAtomicScaleUp,
+		})
+	possibleAtomicScaleUpReq := provreqwrapper.BuildValidTestProvisioningRequestFromOptions(
+		provreqwrapper.TestProvReqOptions{
+			Name:     "possibleAtomicScaleUpReq",
+			CPU:      "100m",
+			Memory:   "1",
+			PodCount: int32(120),
+			Class:    v1beta1.ProvisioningClassAtomicScaleUp,
+		})
+	autoprovisioningAtomicScaleUpReq := provreqwrapper.BuildValidTestProvisioningRequestFromOptions(
+		provreqwrapper.TestProvReqOptions{
+			Name:     "autoprovisioningAtomicScaleUpReq",
+			CPU:      "100m",
+			Memory:   "100",
 			PodCount: int32(5),
 			Class:    v1beta1.ProvisioningClassAtomicScaleUp,
```

Comment:
Let's rename provisioning class in api package

Reply:
I'm not sure I understand what you mean. I don't think we were planning to change API here?